Makefile

```diff
@@ -328,8 +328,8 @@ GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint # TODO Add linting to the GHA also
 
 ## Tool Versions
 CERT_MANAGER_VERSION ?= v1.12.2
-KUSTOMIZE_VERSION ?= v5.0.3
-CONTROLLER_TOOLS_VERSION ?= v0.12.0
+KUSTOMIZE_VERSION ?= v4.5.1
```

Comment:
issue: I have an issue with this, given that 4.5.x is deprecated and 4.5.1 is old version even in 4.x branch and has multiple bugfixes done later. At minimum, this should be 4.5.7.

Users will use 5.x to install our product and now we suddenly don't test it anymore? cass-operator will build structures based on 5.x also. It feels odd if we want to support kustomize installation at all when it doesn't work with the version users would have installed on their machines.

Did you try instead to make it work with 5.x? The current version is 5.4.1, which is quite different from age old 5.0.3.

Reply:
It doesn't work with 5.x. I've updated to 4.5.7 as per your request.